src/pseudopeople/utilities.py

```diff
@@ -192,6 +192,35 @@ def cleanse_integer_columns(column: pd.Series) -> pd.Series:
     return column
 
 
+def number_of_tokens_per_string(s1, s2):
+    """
+    Calculates the number of tokens in each string of a series.
+    s1 is a pd.Series of tokens and we want to see how many tokens exist in each
```

Comment:
nit: "s1 is a pd.Series of tokens and we want to se how many _**of them**_ exist in each ...

Reply:
I've beefed up the docstring.